include/translatable-object.php

```diff
@@ -290,18 +288,19 @@ protected function get_object_terms( array $object_ids, string $taxonomy ) {
 	 *
 	 * @since 3.8
 	 *
-	 * @param int[] $object_ids Array of object IDs to retrieve terms for.
+	 * @param int[]  $object_ids Array of object IDs to retrieve terms for.
+	 * @param string $taxonomy   Taxonomy name.
 	 *
-	 * @return void
+	 * @return int[][]
 	 */
-	private function update_object_term_cache( array $object_ids ): void {
+	private function get_from_object_term_cache( array $object_ids, string $taxonomy ) {
```

Comment:
I suggest to update the method doc description as well.

Reply:
Done in 3f71b14cf8bd51968075031cd3c54d71cd9fb61a